src/include/mpir_objects.h

```diff
@@ -416,6 +416,8 @@ typedef struct MPIR_Object_alloc_t {
     int initialized;            /* */
     void **indirect;            /* Pointer to indirect object blocks */
     int indirect_size;          /* Number of allocated indirect blocks */
+    int num_allocated;          /* Number of allocated objects including both direct and indirect */
+    int num_freed;              /* Number of freed objects including both direct and indirect */
```

Comment:
"freed" implies an action which is confusing. How about `num_avail`?

Reply:
`num_freed` is renamed to `num_avail`.